numba/core/byteflow.py

```diff
@@ -445,6 +445,9 @@ def op_LOAD_GLOBAL(self, state, inst):
     def op_COPY_FREE_VARS(self, state, inst):
         state.append(inst)
 
+    def op_MAKE_CELL(self, state: "State", inst):
```

Comment:
```suggestion
    def op_MAKE_CELL(self, state, inst):
```
not sure this adds anything?

Reply:
oh just to get vscode to provide proper hints at one point. let's remove it.